Canvas/Canvas/CanvasAppDelegate.swift

```diff
@@ -348,11 +347,11 @@ extension AppDelegate: NativeLoginManagerDelegate {
     }
     
     func didLogout(_ controller: UIViewController) {
-        if let entry = Keychain.currentSession {
-            Keychain.removeEntry(entry)
+        if let entry = environment.currentSession {
             environment.userDidLogout(session: entry)
             CoreWebView.stopCookieKeepAlive()
         }
+        Keychain.clearEntries()
```

Comment:
until an app is actually logging in with the new login process, every logout clears all keychain entries.